meetings/2021-08-11.md

```diff
@@ -24,42 +24,50 @@ https://meet.jit.si/solid-authorization
 
 Sarven: To be clear, there is nothing in the WAC spec that prohibits each having their own ACR.
 
-Henry: I'm following the process that has to be followed and writing it to a wiki.
-It's not unworkable; it just has a lot of requests and is a bit inefficient.
+Henry: Indeed. That is the use case I was illustrating it with. Each resource gets its own ACR, but for default behavior the effective ACR is futher up the container hierarchy.
+I'm following the process described in the spec.
+As it is, it's not unworkable; it just has a lot of requests and is very inefficient. 
 Do you disagree that this is inefficient, or have I missed something?
 
 Sarven: I agree that this could be more efficient, but what are you going to adjust?
 
-Henry: My idea is to address it with the minimum changes to WAC.
-Here I can think, for example, of the `rel` link having a mime type `application/trig`, which would be a quad representation. In this case, the server could return a graph with no content in the graph of the resource, but tell you what the default graph is. With one `GET`, you get the default ACL.
+Henry: My idea is to address the problem with the minimum changes to WAC, as always.
+My first thought is that one could have a `rel` type `Link` having a mime type `application/trig`, indicating to the client that a quad representation is available for the ACR. In this case, the server could return a dataset with no content in the graph of the resource, but give information for the default graph. With one `GET`, you get the default ACL.
 
 Sarven: I think I'm a bit confused. There is a slight difference between the spec describing how the default works, and allowing implementations to work up their own implementations. One `GET` to the ACL would give you everything you need to know, as long as the server generates default authorization rules when a resource is created.
 The change that could be on the spec level would be if it prohibits that possibility from happening (omit a representation from an ACL).
 The spec can say all references to an ACL must have a representation.
 Currently, that's a thing we don't enforce.
 
-Henry: There are two answers: either the trig or n3 serialisation would allow us to have the graph of the resource marked empty, and the graph of the default thrown in.
+Henry: 
+The problem is understandable from this dilemna: 
+  - either the `Link: </default.acr>; rel="acl"` points to the effective ACR, but then there is no way for the client to know what the URL of the ACR for that resource would be, where to edit it.
+  - or the `Link: </foo/bar/baz/x.acr>; rel="acl:` point to the ACR of the resource and then we have the problem of the client having to walk down the tree of containers to find the default.
+
+The Trig (or n3) serialisation would allow us to select solution 2, and have the graph of the resource marked empty, and the graph of the default thrown in.
```

Comment:
```suggestion
The TriG (or G3) serialisation would allow us to select solution 2, and have the graph of the resource marked empty, and the graph of the default thrown in.
```

Reply:
Trig is better as you capitalized it by I don't know G3. I guess that is a spell checking error.